File: iot-hub-api/src/main/java/com/soft/iot_hub_api/dto/OverallEquipmentEffectivenesDTO.java
package com.soft.iot_hub_api.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.HashSet;
import java.util.Set;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class OverallEquipmentEffectivenesDTO implements Serializable {

    private Integer oee;
    private Integer availability;
    private Integer performance;
    private Integer quality;

}

File: facke-iot-service/src/facke-iot/domain/overall-equipment-effectivenes.dto.ts
export class OverallEquipmentEffectivenesDto {

  oee: number;
  availability: number;
  performance: number;
  quality: number;

  constructor(overallEquipmentEffectivenesDto: OverallEquipmentEffectivenesDto) {
    this.oee = overallEquipmentEffectivenesDto.oee;
    this.availability = overallEquipmentEffectivenesDto.availability;
    this.performance = overallEquipmentEffectivenesDto.performance;
    this.quality = overallEquipmentEffectivenesDto.quality;
  }
}
